sklearn/svm/tests/test_svm.py

```diff
@@ -996,6 +996,34 @@ def test_ovr_decision_function():
     assert_true(np.all(pred_class_deci_val[:, 0] < pred_class_deci_val[:, 1]))
 
 
+@pytest.mark.parametrize("svc", [svm.SVC, svm.NuSVC])
+def test_svc_ovr_tie_breaking(svc):
+    """Test if predict breaks ties in OVR mode.
+    Related issue: https://github.com/scikit-learn/scikit-learn/issues/8277
+    """
+    X, y = make_blobs(random_state=27)
+
+    xlim = [X[:, 0].min(), X[:, 0].max()]
+    ylim = [X[:, 1].min(), X[:, 1].max()]
+
+    np.random.seed(0)
+    xs = np.linspace(xlim[0], xlim[1], 1000)
```

Comment:
I find it awkward that you make a list above only to index into it here. Why  not do `X[:, 0].min()` etc here?